frontend/packages/console-app/src/components/volume-snapshot/create-volume-snapshot/_create-volume-snapshot.scss

```diff
@@ -0,0 +1,38 @@
+@import '../../../../../console-shared/src/styles/skeleton-screen.scss';
```

Comment:
```suggestion
@import '@console/shared/src/styles/skeleton-screen.scss';
```
Isn't this working?

Reply:
`@console` is not detected in scss files